index.html

```diff
@@ -0,0 +1,900 @@
+                >Her Evangelical Megachurch Was Her World. Then Her Daughter
+                Said She Was Molested by a Minister.</a
+              >
+            </p>
+          </div>
+          <div id="popular-article">
+            <hr />
+            <p>
+              <a
+                href="https://www.nytimes.com/interactive/2019/06/10/magazine/mindy-kaling-late-night-diversity-comedy.html?fallback=0&recId=1MUuhAbut36WTUPj7vgeHMp8Djc&locked=0&geoContinent=EU&geoRegion=IDF&recAlloc=top_conversion&geoCountry=FR&blockId=most-popular&imp_id=240202987&action=click&module=trending&pgtype=Article&region=Footer"
+                >Mindy Kaling on Not Being the Long-Suffering Indian Woman</a
+              >
+            </p>
+          </div>
+          <div id="popular-article">
+            <hr />
+            <p>
+              <a
+                href="https://www.nytimes.com/2019/06/07/dining/one-pot-wonders.html?fallback=0&recId=1MUuhAbut36WTUPj7vgeHMp8Djc&locked=0&geoContinent=EU&geoRegion=IDF&recAlloc=top_conversion&geoCountry=FR&blockId=most-popular&imp_id=507063808&action=click&module=trending&pgtype=Article&region=Footer"
+                >One-Pot Wonders</a
+              >
+            </p>
+          </div>
+          <div id="popular-article">
+            <hr />
+            <p>
+              <a
+                href="https://www.nytimes.com/2019/06/10/sports/warriors-raptors.html?fallback=0&recId=1MUuhAbut36WTUPj7vgeHMp8Djc&locked=0&geoContinent=EU&geoRegion=IDF&recAlloc=top_conversion&geoCountry=FR&blockId=most-popular&imp_id=632138149&action=click&module=trending&pgtype=Article&region=Footer"
+                >How the Warriors Survived Game 5 of the N.B.A. Finals</a
+              >
+            </p>
+          </div>
+          <div id="popular-article">
+            <hr />
+            <p>
+              <a
+                href="https://www.nytimes.com/2019/06/10/us/politics/mueller-judiciary-committee.html?fallback=0&recId=1MUuhAbut36WTUPj7vgeHMp8Djc&locked=0&geoContinent=EU&geoRegion=IDF&recAlloc=top_conversion&geoCountry=FR&blockId=most-popular&imp_id=985745515&action=click&module=trending&pgtype=Article&region=Footer"
+                >Justice Dept. Agrees to Turn Over Key Mueller Evidence to
+                House</a
+              >
+            </p>
+          </div>
+          <div id="popular-article">
+            <hr />
+            <p>
+              <a
+                href="https://www.nytimes.com/2019/06/10/theater/tony-awards-best-worst.html?fallback=0&recId=1MUuhAbut36WTUPj7vgeHMp8Djc&locked=0&geoContinent=EU&geoRegion=IDF&recAlloc=top_conversion&geoCountry=FR&blockId=most-popular&imp_id=918644513&action=click&module=trending&pgtype=Article&region=Footer"
+                >The Best and Worst of the 2019 Tony Awards</a
+              >
+            </p>
+          </div>
+        </div>
+      </aside>
+      <aside id="second-floating-sidebar">
```

Comment:
Here, You don't need 2 aside tags. You can use only one. Delete the one which is causing you troubles.